internal/weldr/api.go

```diff
@@ -1769,16 +1776,33 @@ func (api *API) depsolveBlueprint(bp *blueprint.Blueprint, clean bool) ([]rpmmd.
 			specs[i] += "-*-*.*"
 		}
 	}
+	excludeSpecs := []string{}
+	if outputType != "" {
```

Comment:
How is it possible to reach this function without an image type? I thought depsolv without the set of base packages does not make any sense.

Reply:
I decided to leave this in for now, and rather clean up all the naming in one commit/PR.